app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -13,10 +14,49 @@
   }
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item {
+.gem-c-breadcrumbs--inverse.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item {
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs--collapse-on-mobile .govuk-breadcrumbs__list-item {
+
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
+  @include chevron($direction: "right", $color: govuk-colour("dark-grey"));
```

Comment:
I'm still unsure we need to re-write the arrow/chevron part of the component.

We could do something like this to make the links wrap between mobile and tablet:

```
  @include govuk-media-query($from: mobile, $until: tablet) {
    &:last-of-type {
      position: absolute;
      margin-right: 20px;
    }
  }
```

Reply:
the mixin is so that same code could be used for the "back link" ticket and have same styling as breadcrumb 

https://trello.com/c/ZYJ7nnYt/21-make-back-icon-more-consistent-on-mobiles